derive_client/_bridge/client.py

```diff
@@ -432,7 +437,12 @@ def withdraw_drv(self, amount: int, currency: Currency) -> BridgeTxResult:
             func=[approve_data, bridge_data],
         )
 
-        tx = build_standard_transaction(func=func, account=self.account, w3=context.source_w3, value=0)
+        try:
+            tx = build_standard_transaction(func=func, account=self.account, w3=context.source_w3, value=0)
+        except InsufficientNativeBalance:
+            self._ensure_derive_eth_balance(tx)
+            self.logger.info("Balance top-up triggered; funds are pending. Cannot proceed with tx now.")
+            raise EthGasFundingPending("Awaiting ETH deposit for gas.")
```

Comment:
effectively, this amounts to an another bridge transaction. I made this a raise for now (since won't be there immediately anyway), we can catch that on caller's side. Re-attempting after some time (at least when successful), should work. But what we really want, is to use the exact same logic we use for the bridging of any other asset